client/web/src/components/widgets/inputs/MenuBarInput.vue

```diff
@@ -106,6 +106,7 @@ const menuEntries: MenuListEntries = [
 			[
 				{ label: "Undo", shortcut: ["Ctrl", "Z"], action: async () => (await wasm).undo() },
 				{ label: "Redo", shortcut: ["Ctrl", "⇧", "Z"] },
+				{ label: "Select all layers", shortcut: ["Ctrl", "A"], action: async () => (await wasm).select_all_layers() },
```

Comment:
Can you add a new section between "Document" and "View" called "Layer", and put this in it? Also please use Title Case for "Select All Layers".